src/bundle/Resources/public/webcomponents/ez-main-content.html

```diff
@@ -0,0 +1,85 @@
+<link rel="import" href="../../../assets/ezplatform/polymer/polymer-element.html">
+<!--
+    for extensibility, custom field input need to be loaded before the component
+    that wants to use it. For our own field input, we can import them here
+-->
+<link rel="import" href="../../../assets/ezplatform/ez-field-edit/ez-field-edits.html">
```

Comment:
together with dpobel/ez-field-edit you should remove this import and the following one (as well as the comment above)